test/dag-node-test.js

```diff
@@ -29,10 +29,10 @@ module.exports = (repo) => {
         expect(Buffer.isBuffer(node.data)).to.be.true.mark()
         expect(node.size).to.be.above(0).mark()
 
-        util.serialize(node, (err, serialized) => {
+        DAGNode._serialize(node, (err, serialized) => {
```

Comment:
the serialize and deserialize need to remain at the `util` package, it is the definition of the IPLD Format 

https://github.com/ipld/interface-ipld-format#ipld-format-utils